doc/release-notes.rst

```diff
@@ -286,14 +286,6 @@ Upgrading from Infernalis
   Attempting to create a new version 1 RBD image will result in a warning.
   Future releases of Ceph will remove support for version 1 RBD images.
 
-* The default RBD image features for new images have been updated to enable
```

Comment:
This is duplicated with https://github.com/ceph/ceph/pull/8342/commits/92d1857834108df5a802c34966004bbaffef1a70#diff-d31d1b025d45818c51583cfa5f76c9c9L310